ingestion-core/src/test/java/com/mozilla/telemetry/ingestion/core/transform/PubsubMessageToObjectNodeBeamTest.java

```diff
@@ -0,0 +1,539 @@
+package com.mozilla.telemetry.ingestion.core.transform;
+
+import static org.junit.Assert.assertEquals;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.Field.Mode;
+import com.google.cloud.bigquery.LegacySQLTypeName;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.mozilla.telemetry.ingestion.core.util.Json;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+
+public class PubsubMessageToObjectNodeBeamTest {
```

Comment:
formerly `PubsubMessageToTableRowTest`, but it didn't hit anything specific to ingestion-beam. I moved it to core so that it could access methods visible for testing via package scope rather than public scope.